src/Adapter/Contact/CommandHandler/AddContactHandler.php

```diff
@@ -62,7 +65,16 @@ public function __construct(ValidatorInterface $validator)
     /**
      * {@inheritdoc}
      *
+     * @param AddContactCommand $command
+     *
+     * @return ContactId
+     *
+     * @throws CannotAddContactException
+     * @throws ConstraintDefinitionException
+     * @throws ContactConstraintException
      * @throws ContactException
+     * @throws InvalidOptionsException
```

Comment:
many exceptions like this one are not throws in this function.

Reply:
have changed the PHPStorm setting for call tree analysis depth from 1 to 0. This will limit exceptions of just this function and will not look for them in the inner methods. So in the future, will be less of these to review.